README.md

```diff
@@ -60,6 +63,17 @@ Change the permission to execute:
 chmod a+x ~/.docker/cli-plugins/docker-buildx
 ```
 
+### From `Dockerfile`
+
+Here is how to use buildx inside a Dockerfile through the [`docker/buildx-bin`](https://hub.docker.com/r/docker/buildx-bin) image:
+
+```Dockerfile
+FROM docker
+COPY --from=docker/buildx-bin:latest /buildx /usr/libexec/docker/cli-plugins/docker-buildx
+RUN docker buildx version
+```
```

Comment:
I am not sure how this helps users use `buildx` in dind though?

They would still need to build their own image, no?

Reply:
@maniator Atm yes but could be done in a follow-up on https://github.com/docker-library/docker repo to include this instruction.

Reply:
Shouldnt we also have a PR there in tandem?

Reply:
Could open an an issue [docker-library/docker](https://github.com/docker-library/docker) when this PR is merged as a follow-up if you want yes. Will see what their pov is on the inclusion of buildx in the official image.